### ISSUE TITLE
[macOS Safari + VoiceOver] Up/down arrow key while navigating transcript should narrate

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3764]
## Version

Candidate of `4.12.0`

## Describe the bug

When using macOS Safari + VoiceOver, pressing up/down arrow keys while navigating the transcript, VoiceOver would narrate nothing.

## Steps to reproduce

1. Turn on VoiceOver
2. Talk to MockBot for a few turns
3. Press <kbd>SHIFT</kbd> <kbd>TAB</kbd> a few times to focus on the transcript
4. Press <kbd>UP</kbd> arrow key for a few times

## Expected behavior

While focus is put on activity being focused, VoiceOver narrate nothing.

## Additional context

`aria-descendant` is only supported in a limited number of roles. If the element does not have `role` attribute, VoiceOver will not narrate when `aria-activedescendant` has changed.

[Bug]
